src/components/AvatarGroup/readme.md

```diff
@@ -0,0 +1,460 @@
+        />
+        <CustomText>
+            large
+        </CustomText>
+    </div>
+</div>
+```
+
+##### AvatarGroup
+
+```js
+import React from 'react';
+import { AvatarGroup, Card } from 'react-rainbow-components';
+import styled from 'styled-components';
+
+const columnStyles = {
+    width: '25%',
+    alignItems: 'space-around',
+};
+
+const CustomText = styled.h1.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    color: #a4a7b5;
+    font-size: 14px;
+    ${props =>
+        props.custom &&
+        `
+            font-size: 18px;
+            font-weight: 900;
+
+        `};
+    ${props =>
+        props.variant === 'brand' &&
+        `
+            color: ${props.brand.main};
+        `};
+    ${props =>
+        props.variant === 'success' &&
+        `
+            color: ${props.success.main};
+        `};
+    ${props =>
+        props.variant === 'warning' &&
+        `
+            color: ${props.warning.main};
+        `};
+    ${props =>
+        props.variant === 'error' &&
+        `
+            color: ${props.error.main};
+        `};
+`;
+
+const avatars = [
+    {
+        src:"images/user/user1.jpg",
+        assistiveText:"Jose Leandro",
+        title:"Jose Leandro",
+    },
+    {
+        src:"images/user/user2.jpg",
+        assistiveText:"Tahimi Leon",
+        title:"Tahimi Leon",
+    },
+    {
+        src:"images/user/user3.jpg",
+        assistiveText:"Carlos Miguel",
+        title:"Carlos Miguel",
+    },
+    {
+        src:"images/user/user4.jpg",
+        assistiveText:"Jane Doe",
+        title:"Jane Doe",
+    },
+];
+
+<div className="rainbow-m-around_large">
+    <Card
+        className="rainbow-p-vertical_small rainbow-p-horizontal_large rainbow-flex rainbow-justify_space-around rainbow-m-bottom_small"
+    >
+        <div style={columnStyles}>
+            <CustomText>
+                Organization
+            </CustomText>
+            <CustomText
+                custom
+                variant="brand"
+            >
+                Nexxtway
+            </CustomText>
+        </div>
+        <div style={columnStyles}>
+            <CustomText>
+                Members
+            </CustomText>
+            <AvatarGroup
+                avatars={avatars}
+                maxAvatars={3}
+                showCounter={true}
+                size='small'
+            />
+        </div>
+        <div style={columnStyles}>
+            <CustomText>
+                Localization
+            </CustomText>
+            <CustomText
+                custom
+                variant="brand"
+            >
+                Miami, FL
+            </CustomText>
+        </div>
+        <div style={columnStyles}>
+            <CustomText>
+                Earnings
+            </CustomText>
+            <CustomText
+                custom
+                variant="brand"
+            >
+                $120M
+            </CustomText>
+        </div>
+    </Card>
+    <Card
```

Comment:
I think in order to avoid repeating the markup we should have make this a component